src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -24,17 +24,7 @@
 			</div>
 
 			<div class="columns small-12" v-if="loans.length > 0">
-				<div v-if="!mgCategoryPromoActive" class="loan-card-group row small-up-1 large-up-2 xxlarge-up-3">
```

Comment:
We need to keep this. Otherwise you'll have a promo loan card on non-targeted categories of which there are many.